core/commands/repo.go

```diff
@@ -58,28 +58,20 @@ order to reclaim hard disk space.
 			return
 		}
 
-		gcOutChan, erro := corerepo.GarbageCollectAsync(n, req.Context())
+		gcOutChan := corerepo.GarbageCollectAsync(n, req.Context())
 
-		outChan := make(chan interface{})
+		outChan := make(chan interface{}, len(gcOutChan))
```

Comment:
What does len on a channel tell us? Is it the capacity? or the number of items currently waiting in it?